### PR TITLE
term: DVC-file -> .dvc file from Utkarsh work (2nd chunk)

### PR DESCRIPTION
**Pull request by jorgeorpinel**
> Continuation of https://github.com/iterative/dvc.org/pull/1372#pullrequestreview-425004762

Cc @utkarshsingh99 sorry, I'm basically stealing your work a little here. Need to move things along and the git operations were a little complex.

### CODE REVIEW

File: content/docs/command-reference/remove.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ Remove DVC-tracked files or directories from the <abbr>workspace</abbr>.
 usage: dvc remove [-h] [-q | -v] [-o | -p] [-f] targets [targets ...]
 
 positional arguments:
-  targets        DVC-files to remove.
+  targets        `.dvc` files to remove.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remove.md:11]: @efiop we need to do this same for the core repo when possible please. See #3960

File: content/docs/command-reference/remove.md [hunk 2]
```diff
@@ -8,15 +8,15 @@ Remove DVC-tracked files or directories from the <abbr>workspace</abbr>.
...
 ```
 
 ## Description
 
 This command safely removes data files or directories that are tracked by DVC
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remove.md:16]: Hello again @efiop sorry to pick on you. Will `remove` work only for .dvc files in 1.0? Or also dvc.yaml outputs somehow? I assume only .dvc files.

File: content/docs/command-reference/remove.md [hunk 3]
```diff
@@ -8,15 +8,15 @@ Remove DVC-tracked files or directories from the <abbr>workspace</abbr>.
...
 from the <abbr>workspace</abbr>. It takes a
 [DVC-File](/doc/user-guide/dvc-file-format) as input, removes all of its outputs
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remove.md:18]: Missed this one @utkarshsingh99 

Please find the term non-case-sensitive in all the docs of this PR (and throughout, for later PRs).

File: content/docs/user-guide/basic-concepts/dependency.md [hunk 4]
```diff
@@ -4,5 +4,5 @@ match: [dependency, dependencies]
 ---
 
 A file or directory (possibly tracked by DVC) recorded in the `deps` section of
-a DVC-file (stage file). See `dvc run`. Stages are invalidated when any of their
-dependencies change.
+a `.dvc` file (stage file). See `dvc run`. Stages are invalidated when any of
```
Comment (jorgeorpinel) [inline content/docs/user-guide/basic-concepts/dependency.md:7]: It's dvc.yaml here. There are no deps in .dvc files.
```suggestion
a `dvc.yaml` file. See `dvc run`. Stages are invalidated when any of
```
> Please check formatting though

File: content/docs/user-guide/basic-concepts/dvc-project.md [hunk 5]
```diff
@@ -16,5 +16,5 @@ match:
 Initialized by running `dvc init` in the **workspace** (typically a Git
 repository). It will contain the
 [`.dvc/` directory](/doc/user-guide/dvc-files-and-directories) and
-[DVC-files](/doc/user-guide/dvc-file-format) created with commands such as
+[`.dvc` files](/doc/user-guide/dvc-file-format) created with commands such as
```
Comment (jorgeorpinel) [inline content/docs/user-guide/basic-concepts/dvc-project.md:19]: Should read:

> The .dvc/ directory, as well as dvc.yaml and .dvc files created...

With corresponding files (and formatting), please

File: content/docs/user-guide/basic-concepts/external-dependency.md [hunk 6]
```diff
@@ -3,7 +3,7 @@ name: 'External Dependency'
 match: ['external dependency', 'external dependencies']
 ---
 
-A DVC-file dependency with origin in an external source, for example HTTP, SSH,
-Amazon S3, Google Cloud Storage remote locations, or even other DVC
+A `.dvc` file dependency with origin in an external source, for example HTTP,
```
Comment (jorgeorpinel) [inline content/docs/user-guide/basic-concepts/external-dependency.md:6]: Dependencies only exist in dvc.yaml files actually, so:

```suggestion
A `dvc.yaml` file dependency with origin in an external source, for example HTTP,
```
> Check formatting

File: content/docs/user-guide/basic-concepts/import-stage.md [hunk 7]
```diff
@@ -3,5 +3,5 @@ name: 'Import Stage'
 match: ['import stage', 'import stages']
 ---
 
-Stage (DVC-file) created with the `dvc import` or `dvc import-url` commands.
+Stage (`.dvc` file) created with the `dvc import` or `dvc import-url` commands.
```
Comment (jorgeorpinel) [inline content/docs/user-guide/basic-concepts/import-stage.md:6]:
```suggestion
`.dvc` file created with the `dvc import` or `dvc import-url` commands.
```

File: content/docs/user-guide/basic-concepts/output.md [hunk 8]
```diff
@@ -3,6 +3,6 @@ name: Output
 match: [output, outputs]
 ---
 
-A file or directory tracked by DVC, recorded in the `outs` section of a
-DVC-file. Outputs are usually the result of stages. See `dvc add`, `dvc run`,
+A file or directory tracked by DVC, recorded in the `outs` section of a `.dvc`
+file. Outputs are usually the result of stages. See `dvc add`, `dvc run`,
```
Comment (jorgeorpinel) [inline content/docs/user-guide/basic-concepts/output.md:7]: Should read

> ... section of a dvc.yaml or `.dvc` file. Outputs are usually ...

File: content/docs/command-reference/remove.md (hunk 3 above)
Comment (utkarshsingh99) [reply content/docs/command-reference/remove.md:18]: Okay, I think you have already changed this one.

What about line 18 of this file?
`DVC-file`.
I'm guessing I should change it to `.dvc` file.
I'm not sure how grepping DVC-file missed it though.

File: content/docs/command-reference/remove.md (hunk 1 above)
Comment (efiop) [reply content/docs/command-reference/remove.md:11]: @jorgeorpinel Wait, that looks wrong. It should be "stages or `.dvc` files to remove". It works with `dvc.yaml` too.

File: content/docs/command-reference/remove.md (hunk 2 above)
Comment (efiop) [reply content/docs/command-reference/remove.md:16]: @jorgeorpinel Oh, replied above too. It works on `stages` and `.dvc` files. So yes, it works with `dvc.yaml` too.

File: content/docs/user-guide/basic-concepts/dvc-project.md (hunk 5 above)
Comment (utkarshsingh99) [reply content/docs/user-guide/basic-concepts/dvc-project.md:19]: Should I link all of these `.dvc` file and dvc.yaml to /dvc-file-format?

File: content/docs/user-guide/basic-concepts/dependency.md (hunk 4 above)
Comment (utkarshsingh99) [reply content/docs/user-guide/basic-concepts/dependency.md:7]: I'm assuming you also want all occurrences of `dvc.yaml` file and `.dvc` file to be linked to `/doc/user-guide/dvc-file-format` ?

File: content/docs/command-reference/remove.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/remove.md:11]: I was assuming remove only works with .dvc files like move. OK, great to know, so let's go with this:

```suggestion
  targets        stages (found in `dvc.yaml`) or `.dvc` files to remove.
```

Cc @utkarshsingh99

File: content/docs/command-reference/remove.md (hunk 3 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/remove.md:18]: > What about line 18 of this file?

Yes, that's what I'm talking about. It's DVC-File with capital F (which is wrong BTW). Please grep the term non-case-sensitive in all the docs.

Comment (jorgeorpinel) [reply content/docs/command-reference/remove.md:18]: Also, per https://github.com/iterative/dvc.org/pull/1403#pullrequestreview-426045191 this whole sentence is wrong actually. It should be:

> It takes a stage name (see `-n` option of `dvc run`) or a [`.dvc` file](/doc/user-guide/dvc-file-format) as `target`, removes all of its outputs (`outs` field), and optionally removes the stage entry in [`dvc.yaml`(/doc/user-guide/dvc-file-format) or the `.dvc` file itself.

Cc @efiop is that explanation fully correct? Thanks again.

File: content/docs/command-reference/remove.md (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/remove.md:16]: Moved to https://github.com/iterative/dvc.org/pull/1403#discussion_r437021912

File: content/docs/user-guide/basic-concepts/dependency.md (hunk 4 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/basic-concepts/dependency.md:7]: > I'm assuming you also want all occurrences of dvc.yaml file and .dvc file to be linked to /doc/user-guide/dvc-file-format ?

Correct. Well, not all, just the first ones in each section of a doc. No need to repeat links over and over multiple times per paragraph/section so please check whether they may be linked recently before adding missing links.

File: content/docs/user-guide/basic-concepts/dvc-project.md (hunk 5 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/basic-concepts/dvc-project.md:19]: Not all, just the first ones in each section of a doc. In this case I think yes, link both.

File: content/docs/command-reference/remove.md (hunk 3 above)
Comment (utkarshsingh99) [reply content/docs/command-reference/remove.md:18]: > Please grep the term non-case-sensitive in all the docs

But considering this is just the 2nd chunk, should I restrict the grep search to just the basic concepts folder and the command-reference folder (without the subfolders)?

Comment (jorgeorpinel) [reply content/docs/command-reference/remove.md:18]: Yes but keep this in mind for all the remaining changes in your original branch — should be double checked at some point.

File: content/docs/user-guide/basic-concepts/dependency.md (hunk 4 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/basic-concepts/dependency.md:7]: I see this one is fixed in https://github.com/iterative/dvc.org/pull/1408/files#diff-fae5beb83c8343057fae2259866c8746. Thanks!

File: content/docs/user-guide/basic-concepts/dvc-project.md (hunk 5 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/basic-concepts/dvc-project.md:19]: Moved to #1408

File: content/docs/command-reference/remove.md (hunk 3 above)
Comment (efiop) [reply content/docs/command-reference/remove.md:18]: > Cc @efiop is that explanation fully correct? Thanks again.

@jorgeorpinel It is not if we are talking about 1.0: So `dvc remove ...` in 1.0:

It takes a stage name (see -n option of dvc run) or a .dvc file as target, removes the stage entry in [dvc.yaml(/doc/user-guide/dvc-file-format) or the .dvc file itself and optionally removes the outputs.

**Comment by jorgeorpinel** [comment]
A couple questions to Ruslan are still open but I'll close this now in favor of #1408

File: content/docs/command-reference/remove.md (hunk 3 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/remove.md:18]: Yes, we're talking about 1.0.

Ok thanks Ruslan! Moved this to https://github.com/iterative/dvc.org/pull/1408#pullrequestreview-426658412.

**Comment by jorgeorpinel** [comment]
> couple questions to Ruslan

Actually just https://github.com/iterative/dvc.org/pull/1403#pullrequestreview-426039417 now, and it's more of a comment/reminder, not a question.

Abandoning this ship! ⚓

File: content/docs/command-reference/remove.md (hunk 1 above)
Comment (skshetry) [reply content/docs/command-reference/remove.md:11]: I was thinking of not to mention this yet, only after we release docs for `run`.

Comment (skshetry) [reply content/docs/command-reference/remove.md:11]: `remove` only has `--outs` flag now, and `0.94`'s `remove` and `1.0`'s `remove` behavior are completely different.

Comment (jorgeorpinel) [reply content/docs/command-reference/remove.md:11]: OK, good to know @skshetry . Please open a separate issue or PR from the core repo PR where `remove` is changed (usual process). This PR is about updating "DVC-file" terminology. Thanks

**Pull request closed without merging**